### PR TITLE
Asman umbetov 1 16

### PR DESCRIPTION
**Pull request by fastndead**

### CODE REVIEW

File: tasks/10_remove-whitespaces/remove-whitespaces.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export function trimWhiteSpaces(str){
+    let res = str.split(" ").join('')
```
Comment (vovcyan) [inline tasks/10_remove-whitespaces/remove-whitespaces.js:2]: Use const instead of let in this case

Comment (vovcyan) [inline tasks/10_remove-whitespaces/remove-whitespaces.js:2]: Semicolon missed

File: tasks/11_find-longest-word/find-longest-word.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export function findLongestWord(str){
+    let arrayOfStrings = str.split(" ")
```
Comment (vovcyan) [inline tasks/11_find-longest-word/find-longest-word.js:2]: const and semicolon

File: tasks/11_find-longest-word/find-longest-word.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    return arrayOfStrings.reduce((accamulator, currentValue) => {
+        return accamulator.length < currentValue.length ? currentValue : accamulator
+    }).length
+    
+
```
Comment (vovcyan) [inline tasks/11_find-longest-word/find-longest-word.js:7]: Trailing spaces not allowed

File: tasks/12_title-case/title-case.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    str = str.toLowerCase()
+    let words = str.split(" ")
+    
+    words.forEach((element, index) => {
+        words[index] = element[0].toUpperCase() + element.slice(1)
+    })
+    return words.join(' ')
+  }
+  
+
```
Comment (vovcyan) [inline tasks/12_title-case/title-case.js:11]: Trailing spaces not allowed

**Review by vovcyan — CHANGES_REQUESTED** [review]

File: tasks/10_remove-whitespaces/remove-whitespaces.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+    return res
+}
```
Comment (savchukm) [inline tasks/10_remove-whitespaces/remove-whitespaces.js:4]: Please insert an empty line at the of each file. It is a good practice for programming in general

File: tasks/1_golf-score/golf-score.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+        case par + 2: return GOLF_SCORES.DOUBLE_BOGEY
+        default: if(strokes <= par - 2) {
+                            return GOLF_SCORES.EAGLE 
+                        }else{
+                            return GOLF_SCORES.GO_HOME
+                        }
+    }    
+};
+
+var GOLF_SCORES = {
```
Comment (savchukm) [inline tasks/1_golf-score/golf-score.js:16]: ES6 syntax is better here

File: tasks/5_profile-lookup/profile-lookup.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
+export function lookupProfile(firstName, prop){
+    var foundContact = contacts.find(person => person.firstName === firstName);
+    if(foundContact !== undefined){
```
Comment (savchukm) [inline tasks/5_profile-lookup/profile-lookup.js:3]: Does `foundContact` could have another value like null? What is the purpose of evaluating with undefined exactly?

File: tasks/5_profile-lookup/profile-lookup.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        }
+        else{ 
+            return "No such property"
+        }
+    }else{
+        return "No such contact"
+    }
+}
+
+var contacts = [
```
Comment (savchukm) [inline tasks/5_profile-lookup/profile-lookup.js:15]: ES6

File: tasks/10_remove-whitespaces/remove-whitespaces.js (hunk 1 above)
Comment (savchukm) [reply tasks/10_remove-whitespaces/remove-whitespaces.js:2]: I guess we need to use consistent quotes across the code base. Single quotes is good to see here

File: tasks/15_confirm-ending/confirm-ending.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+export function confirmEnding(str, target) {
+    if(str.substr(-target.length) === target){
+        return true
```
Comment (savchukm) [inline tasks/15_confirm-ending/confirm-ending.js:3]: Returning `true` or `false` directly from function is a bad practice. You can return the result of equal operation

**Review by savchukm — CHANGES_REQUESTED** [review]

**Comment by savchukm** [comment]
Please add one more reviewer at minimum

**Pull request closed without merging**